SampleProject/VOKCoreDataManagerTests/ManagedObjectImportExportTests.m

```diff
@@ -452,7 +452,8 @@ - (void)testFetchWithURI
     person = nil;
     [[[VOKCoreDataManager sharedInstance] managedObjectContext] reset];
 
-    VIPerson *personFromURI = [[VOKCoreDataManager sharedInstance] existingObjectAtURI:uri forManagedObjectContext:nil];
+    VIPerson *personFromURI = (VIPerson *)[[VOKCoreDataManager sharedInstance] existingObjectAtURI:uri
```

Comment:
Actually, looking it over again, I do see the point of the `__kindof` ones.  I don't see the point in returning `VOKManagedObjectSubclass` or having that as a separately-declared type, if `__kindof` only works in collection generics.